### PR TITLE
Add event name to the accessibility label of checkboxes in K5 Schedule

### PR DESCRIPTION
**Pull request by idevainstr**
refs: MBL-15968
affects: Student
release note: none
test plan: see ticket

### CODE REVIEW

**Comment by inst-danger** [comment]
<details><summary>Student.ipa</summary>
  <a href="https://app.bitrise.io/app/88a3641e29e4745c/build/c658be15-8472-4241-b050-e9a5290981e4/artifact/b16764f8ff891022/p/9dabe42dfbd79ac8e87b52d1ff0aa4a3">
    
  </a>
</details>
<details><summary>Teacher.ipa</summary>
  <a href="https://app.bitrise.io/app/88a3641e29e4745c/build/c658be15-8472-4241-b050-e9a5290981e4/artifact/0b6c087ed628dca0/p/0ed17ea57127cb85fdaf91bd6f8f6273">
    
  </a>
</details>
<details><summary>Parent.ipa</summary>
  <a href="https://app.bitrise.io/app/88a3641e29e4745c/build/c658be15-8472-4241-b050-e9a5290981e4/artifact/0b00f27c34f0727f/p/b6b6aed8999176f14b22bbb752e7191a">
    
  </a>
</details>

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 91.82% | 91.82% | -0%

[MBL-15968](https://instructure.atlassian.net/browse/MBL-15968)

#### Affected Apps: Student
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/Core/Dashboard/K5/View/Schedule/K5ScheduleEntryView.swift [hunk 1]
```diff
@@ -153,7 +153,7 @@ public struct K5ScheduleEntryView: View {
                 background.overlay(icon)
             }
         })
-        .accessibility(label: Text("Mark item as done", bundle: .core))
+        .accessibility(label: Text(viewModel.title) + Text(",") + Text("Mark item as done", bundle: .core))
```
Comment (vargaat) [inline Core/Core/Dashboard/K5/View/Schedule/K5ScheduleEntryView.swift:156]: We don't want that comma to get translated so let's use `Text(verbatim: ",")` instead.

**Review by vargaat — CHANGES_REQUESTED** [review]

File: Core/Core/Dashboard/K5/View/Schedule/K5ScheduleEntryView.swift (hunk 1 above)
Comment (vargaat) [inline Core/Core/Dashboard/K5/View/Schedule/K5ScheduleEntryView.swift:156]:
```suggestion
        .accessibility(label: Text(viewModel.title) + Text(verbatim: ",") + Text("Mark item as done", bundle: .core))
```

**Review by vargaat — APPROVED** [review]
QA+1

**Pull request merged**